lib/rules/tscompat.ts

```diff
@@ -263,31 +264,47 @@ export const tscompat = createRule({
 
     return {
       MemberExpression(node) {
+        const checkType = (type) => {
+          const typeName = convertToMDNName(checker, type);
+
+          if (!typeName || typeName === "{}") return;
+
+          // @ts-expect-error It's possible for the property of the member expression to not have a
+          // name. We should test for this. TODO
+          const calleeName = node.property.name;
+
+          const support = findSupport({ typeName, calleeName });
+          const browserTargets = findBrowserTargets(browsers);
+          const failures = getFailures({ support, browserTargets });
+
+          if (failures.length) {
+            const humanReadableBrowsers = formatBrowserList(failures);
+
+            context.report({
+              data: {
+                typeName: `${formatTypeName(typeName)}.${calleeName}()`,
+                browsers: humanReadableBrowsers,
+              },
+              messageId: "incompatable",
+              node,
+            });
+          }
+
+          if (Object.keys(support).length > 0) {
+            // Only check most recently overridden definition
+            return
+          }
+
+          const symbol = type.getSymbol()
+          if (symbol && symbol.getFlags() & (32 | 64 /* SymbolFlags.Class | SymbolFlags.Interface */)) {
```

Comment:
`(SymbolFlags.Class | SymbolFlags.Interface)` works for `npm run build` but doesn't work for `npm run test`.
Might need to dig into `@typescript-eslint/rule-tester`?

Reply:
This was due to CJS/ESM compatibility